sessionctx/variable/session.go

```diff
@@ -474,6 +474,9 @@ type SessionVars struct {
 	//  TxnCtx Should be reset on transaction finished.
 	TxnCtx *TransactionContext
```

Comment:
Will this be removed in the future and all the information comes from the `txnManager` and related `provider`?

Reply:
Yes, I think so. But It seems to be a big work and we need to be patient...